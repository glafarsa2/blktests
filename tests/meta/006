#!/bin/bash
#
# Test a test that fails with bad output which has a very long diff.
#
# Copyright (C) 2017 Omar Sandoval
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

TEST_GROUPS=(meta)

test() {
	for ((i = 1; i <= 100; i++)); do
		echo $i '¯\_(ツ)_/¯'
	done
}
