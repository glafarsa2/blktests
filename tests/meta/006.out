Running meta/006
Test complete
